ribs/visualize.py

```diff
@@ -122,8 +124,8 @@ def grid_archive_heatmap(archive,
             >>> for x in np.linspace(-1, 1, 100):
             ...     for y in np.linspace(-1, 1, 100):
             ...         archive.add(solution=np.array([x,y]),
-            ...                     objective_value=-(x**2 + y**2),
-            ...                     behavior_values=np.array([x,y]))
+            ...                     objective=-(x**2 + y**2),
```

Comment:
Most of these examples still do not work. Can you fix them to call `add` as a batch function in a followup PR?